packages/app/response (2).json

```diff
@@ -0,0 +1 @@
7) 536-1279", "distance": 1391.9180477772838}, {"id": "AEVPmm_ZOTUvFALo1X68pA", "alias": "dutchs-portland", "name": "Dutch's", "image_url": "https://s3-media2.fl.yelpcdn.com/bphoto/P1j_alzwbWjCmuV1bAy9TA/o.jpg", "is_closed": false, "url": "https://www.yelp.com/biz/dutchs-portland?adjust_creative=c32R4bzAw8FryOFaJdPMNQ&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=c32R4bzAw8FryOFaJdPMNQ", "review_count": 319, "categories": [{"alias": "breakfast_brunch", "title": "Breakfast & Brunch"}, {"alias": "sandwiches", "title": "Sandwiches"}], "rating": 4.5, "coordinates": {"latitude": 43.6578360021297, "longitude": -70.260703}, "transactions": ["pickup", "delivery"], "price": "$", "location": {"address1": "28 Preble St", "address2": "", "address3": "", "city": "Portland", "zip_code": "04101", "country": "US", "state": "ME", "display_address": ["28 Preble St", "Portland, ME 04101"]}, "phone": "+12077612900", "display_phone": "(207) 761-2900", "distance": 1333.299695423713}, {"id": "YswZVuJHUvfoF52HcTqs6w", "alias": "the-bakers-bench-westbrook", "name": "The Baker's Bench", "image_url": "https://s3-media1.fl.yelpcdn.com/bphoto/qpCi4lsdq86Iw1V2fMe3yQ/o.jpg", "is_closed": false, "url": "https://www.yelp.com/biz/the-bakers-bench-westbrook?adjust_creative=c32R4bzAw8FryOFaJdPMNQ&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=c32R4bzAw8FryOFaJdPMNQ", "review_count": 101, "categories": [{"alias": "bakeries", "title": "Bakeries"}, {"alias": "sandwiches", "title": "Sandwiches"}, {"alias": "desserts", "title": "Desserts"}], "rating": 4.5, "coordinates": {"latitude": 43.675532, "longitude": -70.366994}, "transactions": ["delivery"], "price": "$", "location": {"address1": "33 Brackett St", "address2": "", "address3": "", "city": "Westbrook", "zip_code": "04092", "country": "US", "state": "ME", "display_address": ["33 Brackett St", "Westbrook, ME 04092"]}, "phone": "+12078567333", "display_phone": "(207) 856-7333", "distance": 7705.2542180996625}, {"id": "uuFQVjP_Ehy8yrn617Vzhg", "alias": "union-bagel-co-portland", "name": "Union Bagel Co.", "image_url": "https://s3-media2.fl.yelpcdn.com/bphoto/BY3FqfrdBhKolov97fHUaA/o.jpg", "is_closed": false, "url": "https://www.yelp.com/biz/union-bagel-co-portland?adjust_creative=c32R4bzAw8FryOFaJdPMNQ&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=c32R4bzAw8FryOFaJdPMNQ", "review_count": 113, "categories": [{"alias": "bagels", "title": "Bagels"}], "rating": 4.0, "coordinates": {"latitude": 43.6630169, "longitude": -70.2530982}, "transactions": [], "price": "$", "location": {"address1": "147 Cumberland Ave", "address2": "", "address3": "", "city": "Portland", "zip_code": "04101", "country": "US", "state": "ME", "display_address": ["147 Cumberland Ave", "Portland, ME 04101"]}, "phone": "+12077474400", "display_phone": "(207) 747-4400", "distance": 1571.324569768622}], "total": 110, "region": {"center": {"longitude": -70.27198791503906, "latitude": 43.66661831770135}}}
```

Comment:
Did you mean to commit this file as well as the other `response.json`? How are they different?

Reply:
I deleted the response (2).json file since it's superfluous at this point. It's saved in postman if I need it.